kmp-launchpad-compose/src/commonMain/kotlin/com/bottlerocketstudios/launchpad/compose/example/navigation/NavigationWrapperExample.kt

```diff
@@ -16,18 +16,18 @@ val exampleNavigationItems = listOf(
         route = "/home",
         selectedIcon = { Icon(imageVector = Icons.Filled.Home, contentDescription = "") },
         unselectedIcon = { Icon(imageVector = Icons.Outlined.Home, contentDescription = "") },
-        label = { Text("Home") },
+        label = { Text("Home") }
     ),
     NavigationItem(
         route = "/settings",
         selectedIcon = { Icon(imageVector = Icons.Filled.Settings, contentDescription = "") },
         unselectedIcon = { Icon(imageVector = Icons.Outlined.Settings, contentDescription = "") },
-        label = { Text("Settings") },
+        label = { Text("Settings") }
     ),
     NavigationItem(
         route = "/profile",
         selectedIcon = { Icon(imageVector = Icons.Filled.VerifiedUser, contentDescription = "") },
         unselectedIcon = { Icon(imageVector = Icons.Outlined.VerifiedUser, contentDescription = "") },
-        label = { Text("Profile") },
+        label = { Text("Profile") }
```

Comment:
Does your linter have trailing commas turned off?  On that note, what's your opinion of trailing commas?

Reply:
I personally have no opinion on them.  It looks like they were removed from KtLint
I added the `.editorconfig` file from the old library that has the trailing comma rule which KtLint uses.